modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl

```diff
@@ -251,6 +257,10 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 		<#assign columnBitmaskEnabled = (entity.finderEntityColumns?size &gt; 0) && (entity.finderEntityColumns?size &lt; 64) && !entity.hasEagerBlobColumn() />
 	</#if>
 
+	<#if !serviceBuilder.isVersionGTE_7_1_0()>
```

Comment:
@yuhai We don't have a method checking if the version is less than 7.1.0, or less than or equals 7.0.0, right?

Reply:
@dantewang, yes, we don't have the method to check.